packages/venia-ui/lib/components/Mask/mask.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
-// import PropTypes from 'prop-types';
-
 import { useStyle } from '../../classify';
 import defaultClasses from './mask.css';
 
+/**
+ * A component that masks content.
+ *
+ * @class Mask
+ * @extends {Component}
```

Comment:
You can remove these lines since it's been refactored into a functional component instead of a class